packages/terra-avatar/src/variants/Avatar.jsx

```diff
@@ -121,5 +128,6 @@ class Avatar extends React.Component {
 
 Avatar.propTypes = propTypes;
 Avatar.defaultProps = defaultProps;
+Avatar.contextType = ThemeContext;
```

Comment:
Do we want to add the context like this?

Reply:
It's the only way to access the new context API with class components aside from using an HOC. It should be fine, up until we need to consume more that one context value here. Then we might want to look into moving this to a hooks-based implementation.

(This will require the jest tests to fully mount the component to populate the context value. Not ideal, but given Avatar's limited depth, shouldn't be a huge deal.)